### PR TITLE
added path support

### PR DESCRIPTION
**Pull request by DvirDukhan**
added path support
add deepequal dependency https://www.npmjs.com/package/deep-equal for easier comparisons.

### CODE REVIEW

File: test/pathBuilder.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Node = require("../src/node");
```
Comment (gkorland) [inline test/pathBuilder.js]:
```
const Node = require("../src/node"),
          Edge = require("../src/edge"),
          Path = require("../src/path");
```

Comment (gkorland) [inline test/pathBuilder.js]: Node = require("../src/node"),
Edge = require("../src/edge"),
Path = require("../src/path");

Comment (gkorland) [inline test/pathBuilder.js]: const Statistics = require("./statistics"),
    Record = require("./record"),
    Node = require("./node"),
    Edge = require("./edge"),
    Path = require("./path"),
    ReplyError = require("redis").ReplyError;

File: test/redisGraphAPITest.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ const assert = require("assert"),
     redis = require("redis"),
     Label = require("../src/label"),
     RedisGraph = require("../src/graph");
+    PathBuilder = require("./pathBuilder");
```
Comment (gkorland) [inline test/redisGraphAPITest.js]: Same here ...

File: src/path.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    getNodes(){
+        return this.nodes;
+    }
+
+    getEdges(){
+        return this.edges;
+    }
+
+    getNode(index){
```
Comment (gkorland) [inline src/path.js]: should you use gettter/setter?

Comment (DvirDukhan) [reply src/path.js]: please elaborate

File: test/pathBuilder.js (hunk 1 above)
Comment (DvirDukhan) [reply test/pathBuilder.js]: fixed

File: test/redisGraphAPITest.js (hunk 2 above)
Comment (DvirDukhan) [reply test/redisGraphAPITest.js]: fixed

File: src/path.js (hunk 3 above)
Comment (DvirDukhan) [reply src/path.js]: it is a property getter, so done, for every zero parametes function

File: src/path.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+"use strict";
+class Path {
+    constructor(nodes, edges){
+        this.nodes = nodes;
+        this.edges = edges;
+    }
+
+    get Nodes(){
```
Comment (gkorland) [inline src/path.js]: @DvirDukhan capital letter?

File: test/pathBuilder.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+"use strict";
+const Node = require("../src/node"),
+    Edge = require("../src/edge"),
+    Path = require("../src/path");
+
+class PathBuilder {
+    constructor(nodeCount){
+        this.nodes = new Array();
```
Comment (gkorland) [inline test/pathBuilder.js]: this.nodes = [];

Comment (DvirDukhan) [reply test/pathBuilder.js]: fixed

File: src/path.js (hunk 4 above)
Comment (DvirDukhan) [reply src/path.js]: changed

**Pull request merged**